### PR TITLE
[strings] fix capitalized second/third word

### PR DESCRIPTION
**Pull request by un1versal**
Most of the newer string are correct but older strings still have this issue.

Only exception is lines like e.g. 148 or 517

May have missed one or other instances. please indicate which lines to fix on review comments.

@MartijnKaijser

### CODE REVIEW

**Comment by MartijnKaijser** [comment]
@jjd-uk are you OK with this?

File: addons/skin.confluence/language/English/strings.po [hunk 1]
```diff
@@ -235,7 +235,7 @@ msgstr ""
 #empty strings from id 31113 to 31115
 
 msgctxt "#31116"
-msgid "Show Recently added Albums"
+msgid "Show recently added Albums"
```
Comment (Jalle19) [inline addons/skin.confluence/language/English/strings.po]: s/Albums/albums/

File: addons/skin.confluence/language/English/strings.po [hunk 2]
```diff
@@ -243,21 +243,21 @@ msgid "Show Recently added Videos"
...
 msgctxt "#31119"
-msgid "Hide Background Fanart"
+msgid "Hide background fanart"
 msgstr ""
 
 #empty strings from id 31120 to 31123
 
 msgctxt "#31124"
-msgid "Show Background \"Now Playing\" Video"
+msgid "Show background \"Now playing\" Video"
```
Comment (Jalle19) [inline addons/skin.confluence/language/English/strings.po]: s/Video/video/

File: addons/skin.confluence/language/English/strings.po [hunk 3]
```diff
@@ -243,21 +243,21 @@ msgid "Show Recently added Videos"
...
 msgstr ""
 
 msgctxt "#31125"
-msgid "Show Background \"Now Playing\" Visualisation"
+msgid "Show background \"Now playing\" Visualisation"
```
Comment (Jalle19) [inline addons/skin.confluence/language/English/strings.po]: s/Visualisation/visualisation/

**Comment by uNiversaI** [comment]
@Jalle19 thx, was unsure about those you mentioned due to context. Better in any case:+1:

**Comment by jjd-uk** [comment]
Looks ok to me

**Pull request merged**